src/mscorlib/src/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -408,7 +408,7 @@ public static void Copy(IntPtr source, IntPtr[] destination, int startIndex, int
         //====================================================================
         public static byte ReadByte([MarshalAs(UnmanagedType.AsAny), In] Object ptr, int ofs)
         {
-            throw new PlatformNotSupportedException();
+            throw new PlatformNotSupportedException(); // https://github.com/dotnet/coreclr/issues/10442
```

Comment:
Any reason not to include a specific message in these?  Can we take the opportunity of a PNSE being thrown to tell the dev getting the exception what they should use instead?